### ISSUE TITLE
Training and Testing Ratio? 

### ISSUE THREAD

**Issue by meharcharansahai** [adeshpande3/Facebook-Messenger-Bot#10]
I am not able to find the dataset division in your code, can you please point m to the snippet where the data division occurs for training, testing and validation. 
I am trying to run this program for Cornell Dataset and need to divide the dataset into separate parts. Also, in the blog posts it's mentioned that we won't need Word2Vec file but I guess the code is not updated to that because word2vec is still required in the seq2seq.py. 
Would be great if you can clear the above points.

**Comment by adeshpande3** [comment]
I didn't do any explicit dataset division in this code. In Seq2Seq.py lines 226-235, I just tested the model on some test strings to inspect the quality of the outputs. That being said, now that I think about it, keeping some sort of validation set probably would have been good to be able to see whether the model starts overfitting, but I was already a little bit low on data, so I just wanted to use it all for training. 

As for the requirement of Word2Vec.py, sorry that's my fault for being unclear. It is still technically needed since we need to obtain a wordList.txt. It is created in the createTrainingMatrices method in word2vec.py. (The embedding matrix is the data structure that isn't needed for seq2seq.py). So, if you don't want to create your own word vectors, then you should only run word2vec.py up to line 77. That way, you'll get the wordList.txt and then you can run seq2seq.py and you should be good. 

I'll make the code changes and the readme changes in a bit. Thanks!

**Comment by adeshpande3** [comment]
Can you check to see whether everything works now?

**Comment by meharcharansahai** [comment]
Yeah, word2vec works as expected with the "y/n" switch. 
I have changed the code to suit the needs of latest Python 3.6 version (some difference in dictionary methods and list access methods), will soon upload the code in fork repo.
Other than that, I will try and incorporate a command line switch to provide a ratio for cross validation.

Thanks. Kudos for this program.
